fixpunkt/fp-newsletter/CVE-2022-47408.yaml

```diff
@@ -0,0 +1,14 @@
+title: 'TYPO3-EXT-SA-2022-017: Multiple vulnerabilities in extension "Newsletter subscriber management" (fp_newsletter)'
+link: https://typo3.org/security/advisory/typo3-ext-sa-2022-017
+cve: CVE-2022-47408
+branches:
+    master:
+        time: 2022-12-13 10:48:00
+        versions: ['>=3.0.0', '<3.2.6']
```

Comment:
this should probably be `>=2.2.0, <3.2.6` instead, to reflect the fact that 2.2+ is also affected (and hasn't been patched until 3.2.6)

Reply:
ok, looking at packagist, there is no releases in the 2.2.1-2.4.0 range, so why is the advisory talking about them ?

Reply:
I will update version constraints as suggested.

Version 2.4.0 seems to be a release, that has never been tagged and published to packagist. It has however been uploaded to TYPO3 extension repository https://extensions.typo3.org/extension/fp_newsletter as zip-package.